apps/viewer/src/components/viewer-video-view/index.tsx

```diff
@@ -75,19 +75,21 @@ const ViewerVideoView = ({
   };
 
   const handleChangeVolume = (newVolume: number) => {
-    if (newVolume === 0) {
-      audioTrack.enabled = false;
-
+    if (newVolume === 0 && audioEnabled) {
```

Comment:
Is the `newVolume` has higher priority than `audioEnabled`? If true, we could remove `audioEnabled` here.
```suggestion
    onToggleAudio?.(newVolume > 0)
```